pkg/status/status.go

```diff
@@ -78,6 +80,10 @@ func GetStatus() (map[string]interface{}, error) {
 		httputils.NoProxyWarningMapMutex.Unlock()
 	}
 
+	if config.IsKubernetes() {
+		stats["autodiscoveryErrors"] = common.AC.GetAutodiscoveryErrors()[names.Kubernetes]
```

Comment:
Even though only the kubelet errors are supported for now, I think we should check all provider errors returned by `GetAutodiscoveryErrors` to fill `stats["autodiscoveryErrors"]`